ansible/roles/machine/setup/tasks/install_packages.yml

```diff
@@ -49,12 +49,12 @@
 - name: install py3 pip dependencies
   pip:
     executable: pip3
-    name: "{{ python_packages_to_install | join(' ') }}"
```

Comment:
Will this change work well with f28?

Reply:
Yes, it should, this change impacts ansible, specific pip module, according to this doc:

[Link to Ansible doc](https://docs.ansible.com/ansible/2.7/modules/pip_module.html)

```
The name of a Python library to install or the url(bzr+,hg+,git+,svn+) of the remote package.
This can be a list (since 2.2) and contain version specifiers (since 2.7).
```
According to [Ansible Release on GitHub](https://github.com/ansible/ansible/releases?after=v2.1.5.0-1) We can have an idea about the release date of 2.2, which is early 2017.

Also, due to #236, we will need to generate new boxes which will include latest versions of Ansible on it.

Reply:
Hey @f-trivino I've confirmed:

- This playbook I've applied this changes is only used when generating a template box. Since we can run the [create-template-box sh](https://github.com/freeipa/freeipa-pr-ci/blob/master/create-template-box) in any machine to create a new template box, we only need to ensure that this fedora running create-template-box have at least Ansible 2.2 version.

- During the execution of tests, the Vagrant uses as provisioner the [machine/provision](https://github.com/freeipa/freeipa-pr-ci/blob/b102ea6d6f3157e45303d66a3a2d4e6c5784ceee/templates/vagrantfiles/Vagrantfile.master_1repl_1client#L42), thi [provision](https://github.com/freeipa/freeipa-pr-ci/blob/master/ansible/provision.yml) calls the machine/provision.

- So, this change should not impact the the tests :+1:

Reply:
Also, PR-CI requirements section on [README](https://github.com/freeipa/freeipa-pr-ci/blob/4d68720f13fbf6993bd3f15d919428f4774890b9/README.md#other):

> #### Other
> 
> - `ansible >= 2.3` on the control machine
> - GitHub token with *Full control of private repositories* (Settings ->
>   Personal access tokens)
> - `freeipa_pr_ci` private key placed in `keys/`
> - Cloud `config` and `credentials` files in `keys/`
> - private keys in `keys/` have permissions set to 0600
>

Reply:
ok!